packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -126,18 +126,27 @@ export default function useTdpClientCanvas(props: Props) {
     }
   };
 
+  // Not used for DesktopSessions (but needed for playback)
+  const onClientScreenSpec = () => {};
+
+  // Block browser context menu so as not to obscure the context menu
+  // on the remote machine.
+  const onContextMenu = () => false;
```

Comment:
I'm not sure I follow why we need these, and why it needs to be defined here

Reply:
`onClientScreenSpec` is a holdover from before I took Yassine's advice and made that an optional prop.

`onContextMenu` is needed so that the browser's context menu doesn't pop up when the user right clicks on the canvas (`TdpClientCanvas`) -- in that case, only the _Windows_ context menu should show up and shouldn't be blocked by the browser's menu.